### PR TITLE
Cover only grid added to library 

### PR DESCRIPTION
**Pull request by micaelagimenez**
Resolves [#6517 ](https://github.com/tachiyomiorg/tachiyomi/issues/6517)

cover grid

### CODE REVIEW

**Comment by vengiare** [comment]
Does it address the problem mentioned in the previous closed issue (#3272) about the same feature? Or is it not that much of a problem now?

> there's occasional issues with covers not showing up properly

**Comment by micaelagimenez** [comment]
@vengiare I hadn't seen that issue (#3272) but I didn't notice any issue myself with the covers when I tried this feature.

**Comment by vengiare** [comment]
Ideal solution for that problem would be to show title when cover hasn't loaded yet, but that could be complicated.

**Comment by Horodep** [comment]
> Ideal solution for that problem would be to show title when cover hasn't loaded yet, but that could be complicated.

I may say some nonsense but doesn't changing the order layer solves the issue? 
I mean.. we can always draw text, but then draw cover over the text. If there is no cover, the text won't be hidden. And there will be no complicated bugs about determining if cover if displayed or not.

**Comment by micaelagimenez** [comment]
I added a new commit with an else added to the holder. This way, in case the image is null or empty, the title is displayed.

**Comment by arkon** [comment]
Does this actually make any sense to have when browsing?

**Comment by FourTOne5** [comment]
Shouldn't "Cover Only Grid" sound better?

**Comment by CarlosEsco** [comment]
> Does this actually make any sense to have when browsing?

Not at all.  Should be a library only thing

**Comment by arkon** [comment]
@micaelagimenez Do you mind updating it so it's only an option in the library?

**Comment by micaelagimenez** [comment]
Sure! I can do that and change the name if everyone prefers

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -222,7 +222,7 @@
     <string name="action_display">Mostrar</string>
     <string name="action_display_grid">Rejilla compacta</string>
     <string name="action_display_list">Lista</string>
-    <string name="action_display_no_title_grid">Rejilla sin títulos</string>
+    <string name="action_display_cover_only_grid">Rejilla con portada</string>
```
Comment (FourTOne5) [inline app/src/main/res/values-es/strings.xml:225]: Only add `en` strings others are translated and added via weblate

Comment (FourTOne5) [inline app/src/main/res/values-es/strings.xml:225]: Only add `en` strings others are translated and added via weblate

**Comment by Horodep** [comment]
> No title grid renamed and now only available in library
You forgot to rename PR itself.

**Pull request merged**